R/base_bind_rows.R

```diff
@@ -0,0 +1,38 @@
+# This function was created to standardize column names between two databases 
+  # and then base::rbind the new df (the function doesn't work with diff cols)
+
+# Steps:
+  # 1. Get set diff of col names, change `.` to `_` in df col names 
+  # 2. Add missing cols to each data frame by using a setdiff on col names
+  # 3. Use base::rbind to combine dfs, which now have same cols (filled with NA)
+
+base_bind_rows <- function(df1, df2) {
```

Comment:
Thanks for this! 
Cool stuff, but I am unsure about using `data.frame` for storing compilation logs. One argument is that different reports can use different params, which will result in a very sparse data table once in `csv`. A `list` structure might be simpler, e.g. for a single log item:

```r
$source
[1] "toto_2019-01-01.Rmd"

$params
$params$sc
[1] "beni"

$params$epicasedef
[1] "confirmed" "probable" 

$params$time_stamp
[1] "2019-10-14 16:01:20 CAT"

$output
[1] ... # whatever file name was generated

```

Reply:
updated in the latest branch, now an RDS file... fine, it's better...

This stores the args (including from ...), outputs, and file name by date:

```r
> log_file$foo$`2019-10-16 17:34:44`$file
[1] "foo_2018-06-29.Rmd"


> log_file$foo$`2019-10-16 17:34:44`$output_files[[1]]
[1] "/private/var/folders/78/j2tl5py10rq0mkyl34b1kg7m0000gn/T/RtmpzG0uVt/factory_YExqeWLMPiHn/report_outputs/foo_2018-06-29/compiled_other_test_2019-10-16_17-34-44/figures/boxplots-1.pdf"

> log_file$foo$`2019-10-16 17:51:13`$params
$other
[1] "two"

$more
$more$thing
[1] "foo"


$quiet
[1] TRUE
```
Having a datetime stamp that starts with a number isn't ideal, but i'm guessing we'll probably just be accessing the last one in each report type. I wanted to keep the full timestamp in case a report was run more than once in a day (though we could just keep the last one only

Reply:
@thibautjombart the requested change is ready for review when you have a chance